getting_started/setup_vm/app-dev.yml

```diff
@@ -2,6 +2,7 @@
   vars:
     run_only: false
     platform: "sgx"
+    clang_version: "10"
```

Comment:
Should the appropriate clang version should be passed in [here](https://github.com/microsoft/CCF/blob/main/.github/workflows/containers.yml) too? i.e. 12 for virtual/snp and 10 for SGX?

Reply:
I would prefer not to make the switch this time to minimise the gap between the tag and the release containers. We can switch in dev6.